_posts/2016-05-18-mid-may-features-update.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: post
+title: "Mid-May Features Update"
+author: "Hayley MacLeod"
+categories: pitchplay
+---
+
```

Comment:
Should have some kind of introductory text before the first header break. The `Do you friends practice?` stuff below kind of acts like that, but it feels a little forced.